packages/cli/src/commands/start.ts

```diff
@@ -196,8 +198,8 @@ export default class Start extends Command {
       min_uptime: 10000,
       kill_timeout: 2000,
       args,
-      error: join(Config.logsPath, `${environmentInfo.name}-error.log`),
-      output: join(Config.logsPath, `${environmentInfo.name}-out.log`),
+      error: environmentInfo.disableLogToFile ? '/dev/null' : join(Config.logsPath, `${environmentInfo.name}-error.log`),
+      output: environmentInfo.disableLogToFile ? '/dev/null' : join(Config.logsPath, `${environmentInfo.name}-out.log`),
```

Comment:
Using '/dev/null' should work with PM2 according to their [documentation ](https://pm2.keymetrics.io/docs/usage/log-management/#disable-logging)

Reply:
Using `/dev/null` does not create a file on windows on my machine when using PM2.

Reply:
yes, it doesn't, it works perfectly. I added this comment in case of doubt, because in the other remark on `fileTransport` using `/dev/null` doesn't work. 
I didn't want you to look into this one also. But I guess my comment was rather confusing... Sorry!

Reply:
Looks like the interface for this accepts either a string path or undefined, so if we don't want error or output to a file, then we provide undefined, not the prettiest approach, but it seems it will still log to file /dev/null on windows

Reply:
According to their to [their documentation](https://pm2.keymetrics.io/docs/usage/log-management/#disable-logging) it either accept `/dev/null` or `NULL` as strings. 
Something like: `error: '/dev/null'`. What you did was correct, as they won't use the path directly.